sdk/src/main/java/com/meniga/sdk/converters/MenigaTransactionUpdateConverter.java

```diff
@@ -0,0 +1,75 @@
+package com.meniga.sdk.converters;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.reflect.TypeToken;
+import com.meniga.sdk.helpers.GsonProvider;
+import com.meniga.sdk.models.transactions.MenigaComment;
+import com.meniga.sdk.models.transactions.MenigaTransaction;
+import com.meniga.sdk.models.transactions.MenigaTransactionPage;
+import com.meniga.sdk.models.transactions.MenigaTransactionUpdate;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.RequestBody;
+import okhttp3.ResponseBody;
+import retrofit2.Converter;
+import retrofit2.Retrofit;
+
+/**
+ * Copyright 2017 Meniga Iceland Inc.
+ * Created by agustk on 16.2.2018.
+ */
+public class MenigaTransactionUpdateConverter extends MenigaConverter {
+    @Override
+    public Converter<ResponseBody, ?> responseBodyConverter(Type type, Annotation[] annotations, final Retrofit retrofit) {
+        Type typeOfTransactionUpdate = new TypeToken<MenigaTransactionUpdate>() {}.getType();
+
+        if (typeOfTransactionUpdate.equals(type)) {
+            return new Converter<ResponseBody, Object>() {
+                @Override
+                public Object convert(ResponseBody resBody) throws IOException {
+                    Gson gson = GsonProvider.getGsonBuilder();
+
+                    InputStreamReader isr = null;
+                    try {
```

Comment:
You can use try-with-resources here:

```
try (InputStreamReader inputStream = new InputStreamReader(responseBody.byteStream())) {
  ...
}
```

Reply:
We need to support min sdk v. 14 for bureaucratic reasons.